stylesheets/commons/Panels.less

```diff
@@ -27,3 +27,71 @@ Author(s): FO-nTTaX
 .panel-box-body {
 	padding: 10px;
 }
+
+/* Temp, for testing purposes */
+
+.dota2-test {
+	.panel-box {
+		box-shadow: 0 0.0625rem 0.25rem 0 rgba( 0, 0, 0, 0.12 );
+		border: 0;
+		border-radius: 0.5rem;
+
+		&.is--collapsed {
+			.panel-box-heading {
+				border-radius: 0.5rem;
+
+				&-icon {
+					transform: rotate( 180deg );
+				}
+			}
+
+			.panel-box-body {
+				display: none;
+			}
+		}
+	}
+
+	.panel-box-heading {
+		display: flex;
+		align-items: center;
+		background-color: var( --clr-cinnabar-30 );
```

Comment:
it probably should have a Bruinen fallback

Reply:
I'm going to create a new variable in LakesideView and Bruinen to get the correct color. Because `clr-wiki-primary` is not returning the color that is used in the design. And I don't want to overwrite `clr-wiki-primary` because this is used in other classes.

Reply:
This has been resolved by adding `--clr-wiki-theme-primary` to LakesideView. For Bruinen we can use the `--wiki-color-medium`.